primitives/float.fil

```diff
@@ -119,14 +119,14 @@ comp FAdd[WIDTH, EXPONENT, MANTISSA]<'G: 1> (
 
     // renormalization
     bundle shift_amts[MANTISSA+1]: for<i> ['G, 'G+1] WIDTH;
-    _mant_1_const := new Const[WIDTH, MANTISSA+2]<'G, 'G+1>(); // constant value set if all mantissa bits are 0
```

Comment:
Might be worth giving local names to terms like `MANTISSA+2`. Also, in a follow up PR, we should rename things to follow the naming convention (#292)